types/jest/index.d.ts

```diff
@@ -29,8 +29,6 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
 
-/// <reference types="jest-diff" />
```

Comment:
I added this in #33030 due to a bug in the publisher meaning it wasn't added as a dependency. However, it was not removed when `@types/jest-diff` was deprecated in #39589.